ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/ui/Pages.java

```diff
@@ -2,8 +2,10 @@
 
 import com.oviva.ehealthid.fedclient.IdpEntry;
 import java.util.Comparator;
+import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
+import java.util.Locale;
+import java.util.ResourceBundle;
 
 public class Pages {
```

Comment:
Instead of passing a language and hoping for the best, why not pass the list of preferred languages and negotiate based on the available bundles?
Then you don't have to hardcode the supported languages anywhere.
Maybe you also want to centralize the bundle + language management since you need it here as well as for exceptions apparently.